templates/application.template.yml

```diff
@@ -0,0 +1,57 @@
+hedera:
+  mirror:
+    grpc:
+      listener:
+        type: SHARED_POLL
+    importer:
+      parser:
+        record:
+          entity:
+            redis:
+              enabled: false
+          sidecar:
+            enabled: {{sidecarEnabled}}
+      downloader:
+        accessKey: minioadmin
+        cloudProvider: "S3"
+        secretKey: minioadmin
+        bucketName: hedera-streams
+        endpointOverride: http://minio:9000
```

Comment:
Can you add here `timeout: {{timeout}}` ? Default is 30 . Indentation is the same as `endpointOverride`, for example.